src/hooks/withIAPContext.tsx

```diff
@@ -6,6 +6,7 @@ import {
   promotedProductListener,
   purchaseErrorListener,
   purchaseUpdatedListener,
+  clearTransactionIOS,
```

Comment:
You don't need to edit the withIapContext file, the function can be imported straight away in the useIAP hook similar as `finishTransaction` and `finishTransaction as iapFinishTransaction`

Reply:
true 👍